### PR TITLE
add git ignore & java guides link for cn infomation.

### PR DESCRIPTION
**Pull request by chenmudu**
add java guides to otel docs-cn.

### CODE REVIEW

**Review by WPH95 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@
 - [技术标准](./specification/Readme.md) 
 - [社区入口](community/Readme.md)
 - [快速入门](QUICKSTART.md)
+    - [上手指南之Java相关](https://github.com/chenmudu/open-telemetry-java-guides)
```
Comment (WPH95) [inline README.md:10]:
```suggestion
    - [上手指南之 Java 相关](https://github.com/chenmudu/open-telemetry-java-guides)
```

**Review by WPH95 — CHANGES_REQUESTED** [review]

**Comment by WPH95** [comment]
personally. I don't recommend accepting unofficial documents for doc-cn at this time

**Comment by tydhot** [comment]
> personally. I don't recommend accepting unofficial documents for doc-cn at this time

I think there should be an index of related article  to put some unofficial documents

**Comment by chenmudu** [comment]
> > personally. I don't recommend accepting unofficial documents for doc-cn at this time
> 
> I think there should be an index of related article to put some unofficial documents

Maybe these unofficial projects should have a citation and eye-catching list. I just want to provide some convenience for more people. Hope this is a good and sincere suggestion. thanks a lot.

**Comment by sunface** [comment]
@WPH95 @tydhot 

From my perspective, there should be no difference between official and unofficial at early stage, we need to encourage more people to contribute :D

**Comment by GlowingRuby** [comment]
> > personally. I don't recommend accepting unofficial documents for doc-cn at this time
> 
> I think there should be an index of related article to put some unofficial documents

I agree with that. Unofficial documents should be welcomed, but there should be a clear distinction between them and official documents.

**Comment by chenmudu** [comment]
@sunface @WPH95 @GlowingRuby 

So, should we add a hands-on guide(cn eg. 上手指南) to the first-level list of read me file contents? Some related items should be listed in the second list, along with their source, language and whether they are official.(cn eg. Java指南(unofficial)).

Hope to reach a consensus with all as soon as possible. Thanks a lot.

**Review by sunface — APPROVED** [review]

**Pull request merged**